src/features/validation/validate.ts

```diff
@@ -208,8 +208,24 @@ export class ApplyMarkersCommand extends FeedbackCommand {
     }
 }
 
-function addCSSClassToIssueParent(modelElement: SParentElement, issueMarker: SIssueMarker) {
-    addCssClasses(modelElement, [getSeverity(issueMarker)]);
+function addMaxSeverityCSSClassToIssueParent(modelElement: SParentElement, issueMarker: SIssueMarker) {
+    const severityCSSClass = getSeverity(issueMarker);
+
+    if (modelElement.cssClasses === undefined) {
```

Comment:
Your can ommit the  `=== undefined` here by using `if (!modelElement.cssClasses)`